app/models/mixins/miq_web_server_worker_mixin.rb

```diff
@@ -175,14 +185,15 @@ def self.reserve_port(ports)
     end
 
     def command_line_params
-      params = {}
-      params[:port] = self.port if self.port.kind_of?(Numeric)
+      params = super
+      params[:Port] = self.port if self.port.kind_of?(Numeric)
       params
     end
 
     def start
       delete_pid_file
       ENV['PORT'] = self.port.to_s
+      ENV['MIQ_GUID'] = self.guid
```

Comment:
Need to review this to see if these ENV variables are needed anymore.  More deletes?